### DISCUSSION TITLE
Select Component how to get value

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#3557] [Q&A]
Select Component Data is 
data={[
        { value: '1', label: 'React' },
        { value: '2', label: 'Angular' },
        { value: '3', label: 'Svelte' },
        { value: '4', label: 'Vue' },
      ]}
      
how to get value on select

**Comment by rtivital** [comment]
https://mantine.dev/core/select/#controlled

**Reply by goodlythink** [reply]
thank.
